design-toolkit/components/src/components/date-field/styles.ts

```diff
@@ -39,6 +39,7 @@ export const DateFieldStyles = tv({
     error: 'fg-serious text-body-xs empty:hidden',
     dateSegment:
       'focus-visible:bg-highlight focus-visible:text-inverse-light focus-visible:outline-none',
+    monthSegment: 'inline-block w-[4ch] text-right',
```

Comment:
Shouldn't it be 3 characters? What made you choose 4?

Reply:
I think that might be because you don't have the same font settings on the thing that's doing the sizing. If the sizing was on the input, I think it would work. Though it may also require taking into account letter spacing